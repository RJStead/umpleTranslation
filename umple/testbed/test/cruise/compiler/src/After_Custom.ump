generate Java "../src-gen-umple";

class AfterCustom
{
    after custom foo {
        a = 1;
        b = a;
    }
    
    public void foo() {
        int a = 0;
        a++;
    }
}
