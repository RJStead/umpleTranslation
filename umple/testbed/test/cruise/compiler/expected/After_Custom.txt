After_Custom.ump:7: error: cannot find symbol
            b = a;
            ^
  symbol:   variable b
  location: class AfterCustom
1 error